packages/graph-engine/src/nodes/string/join.ts

```diff
@@ -18,24 +17,26 @@ export default class NodeDefinition extends Node {
   }>
   constructor(props: INodeDefinition) {
     super(props);
-    this.addInput("array", {
-      type: arrayOf(StringSchema),
-      visible: true,
+    this.addInput("items", {
```

Comment:
Lets not do this. This forces a user to connect multiple individual connections when they might already have a single array value. I'd like to keep variadic only for special cases like arrify.

Plus this will break existing graphs